app/views/devise/passwords/new.html.haml

```diff
@@ -9,6 +9,6 @@
     = f.email_field :email, class: 'form-control', required: 'true', autofocus: true
 
   .col-sm-8.field
-    = f.submit "Send me reset password instructions", class: 'btn', style: 'margin-top: 10px;', disable_with: "Sending..."
+    = f.submit "Send me reset password instructions", class: 'btn btn-primary', style: 'margin-top: 10px;', disable_with: "Sending..."
```

Comment:
Another place to use `…`

Reply:
Grepped through and replaced a bunch of them, thanks.